src/functionaltest/java/com/ericsson/ei/config/Utils.java

```diff
@@ -0,0 +1,117 @@
+package com.ericsson.ei.config;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Properties;
+
+import org.apache.commons.compress.archivers.tar.TarArchiveEntry;
+import org.apache.commons.compress.archivers.tar.TarArchiveInputStream;
+import org.apache.commons.compress.compressors.bzip2.BZip2CompressorInputStream;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.IOUtils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Utils {
+    private static final Logger LOGGER = LoggerFactory.getLogger(Utils.class);
+
+    /**
+     * Loads properties from file path.
+     *
+     * @param filePath
+     *            path to the properties file
+     * @return properties object
+     */
+    public static Properties getProperties(final String filePath) {
+        Properties properties = new Properties();
+        File file = new File(filePath);
+        try {
+            properties.load(new FileInputStream(file));
+        } catch (IOException e) {
+            LOGGER.error("Failed load properties from path.\nError: {}", e.getMessage());
+        }
+        return properties;
+    }
+
+    /**
+     * Extracts a BZip2 archive to a destination folder.
+     *
+     * @param firefoxTarballFilePath
+     *            string containing the firefox BZip2 filepath.
+     * @param destination
+     *            string containing a destination path.
+     */
+    public static void extractBZip2InDir(final String firefoxBZip2FilePath, String destinationPath) {
+        LOGGER.info("Extracting firefox BZip2 archive...");
+        try (TarArchiveInputStream fileInput = new TarArchiveInputStream(
+                new BZip2CompressorInputStream(new FileInputStream(firefoxBZip2FilePath)))) {
+            TarArchiveEntry entry;
+            try {
+                while ((entry = fileInput.getNextTarEntry()) != null) {
+                    if (entry.isDirectory()) {
+                        continue;
+                    }
+                    final File curfile = new File(destinationPath, entry.getName());
+                    final File parent = curfile.getParentFile();
+                    if (!parent.exists()) {
+                        parent.mkdirs();
+                    }
+                    final FileOutputStream fileOutput = new FileOutputStream(curfile);
+                    IOUtils.copy(fileInput, fileOutput);
+                    fileOutput.close();
+                }
+            } catch (IOException e) {
+                LOGGER.error("IOException.\nError: {}", e.getMessage());
+            }
+            fileInput.close();
```

Comment:
this fileInput.close() maybe should be in a finally so wven if it get an error it closes the file.

Reply:
Removing the close completely, see https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html